Misc/NEWS.d/next/Build/2022-05-25-13-56-00.gh-issue-93207.B9Rubf.rst

```diff
@@ -0,0 +1 @@
+Remove configure check for ``stdarg.h`` as it was not required. Patch by Kumar Aditya.
```

Comment:
I suggest to rephrase it differently:

va_start() with two parameters, like ``va_start(args, format)``, is now required to build Python. va_start() is no longer called with a single parameter.

I suggest to copy this entry to the What's New in Python 3.12 document, in the Build Changes.

Reply:
done.